srcpkgs/python3-signedjson/template

```diff
@@ -1,15 +1,20 @@
 # Template file for 'python3-signedjson'
 pkgname=python3-signedjson
-version=1.1.0
-revision=4
-wrksrc="python-signedjson-${version}"
+version=1.1.1
+revision=1
+wrksrc="signedjson-${version}"
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
 depends="python3-canonicaljson>=1.0.0 python3-unpaddedbase64>=1.0.1
```

Comment:
```suggestion
depends="python3-canonicaljson python3-unpaddedbase64
```

Reply:
Incorporated, thanks. That was the first time I've seen versions explicitly included so I wasn't sure what to make of it.